src/components/TextField/TextField.tsx

```diff
@@ -400,6 +420,15 @@ class TextField extends React.PureComponent<CombinedProps, State> {
     onChange(String(newValue.toFixed(decimalPlaces)), this.state.id);
   };
 
+  private handleClearButtonPress = () => {
+    const {
+      state: {id},
+      props: {onClearButtonClick},
+    } = this;
+
+    onClearButtonClick && onClearButtonClick(id);
```

Comment:
It's used here.

We can also avoid an early return (and testing for that early return) by simply short circuiting if there is no handler provided.